CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ endif()
 
 # Boost for CRC calculator, Test, and Logging
 find_package(Boost COMPONENTS log unit_test_framework REQUIRED)
+if(Boost_FOUND)
+    include_directories(${Boost_INCLUDE_DIRS})
+endif()
```

Comment:
Is this necessary? (I don't think it should be. We should be "linking" the Boost target to our targets which includes the necessary directories as well as links the libraries.)

Reply:
Ah sorry this is a leftover from when I was trying to compile on my mac... Got rid of it